styles/colors.less

```diff
@@ -222,6 +223,10 @@
 @sand-formcheckboxitem-disabled-text-color: inherit;
 @sand-formcheckboxitem-disabled-focus-text-color: @sand-formcheckboxitem-disabled-text-color;
 
+// Header
+// ---------------------------------------
+@sand-header-text-shadow: 6px 12px 12px rgba(0, 0, 0, 0.3);
```

Comment:
I guess we could replace `rgba(0, 0, 0, 0.3)` with `@sand-shaodw-color`

Reply:
It's replaced.